homeassistant/components/ialarm/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "ialarm",
+  "name": "Antifurto365 iAlarm",
+  "documentation": "https://www.home-assistant.io/integrations/ialarm",
+  "requirements": [
+    "pyialarm==1.3"
+  ],
+  "codeowners": [
+    "@RyuzakiKK"
+  ],
+  "config_flow": true
```

Comment:
This is probably discoverable via dhcp

https://developers.home-assistant.io/docs/creating_integration_manifest/#dhcp

Reply:
I added the dhcp discovery. Unfortunately the device doesn't provide an hostname, so I had to use only the MAC address.

Probably that's fine, but if this is too broad I can remove the dhcp discovery.

Reply:
`async_step_dhcp` needs to be implemented for this to pass the ip to the integration.

Its ok to remove it for now and add it in a future PR

Reply:
Okay, I removed it so that the merge of this integration can be faster. And after it will be included in HA I can create a separate PR to add the dhcp discoverability.